docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -95,18 +92,19 @@ az ls-groups
 
 ## Create an alias command with arguments
 
-You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the braces is required.
```

Comment:
My understanding is that Jinja2 treats interior whitespace as significant. All of their documentation uses whitespace inside of brackets. If this is the case, this line should be left in.

From experience with documentation, not calling out where whitespace is significant as part of a spec can lead to user confusion and frustration, as whitespace is often not expected to be significant in a tokenizer where there is a clear delineation between alphanumeric and nonalphanumeric tokens.